Source/Relativity.Transfer.SDK.Samples.Repository/BearerTokenAuthentication.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+using Relativity.Transfer.SDK.Interfaces.Paths;
+using Relativity.Transfer.SDK.Samples.Core.Attributes;
+using Relativity.Transfer.SDK.Samples.Core.Authentication;
+using Relativity.Transfer.SDK.Samples.Core.Configuration;
+using Relativity.Transfer.SDK.Samples.Core.Helpers;
+using Relativity.Transfer.SDK.Samples.Core.Runner;
+
+namespace Relativity.Transfer.SDK.Samples.Repository;
+
+[Sample(1, "A bearer token authentication",
+    "The sample illustrates how to implement a bearer token authentication in RelativityOne.",
+    typeof(BearerTokenAuthentication),
+    TransferType.UploadFile)]
+internal class BearerTokenAuthentication(
+    IConsoleLogger consoleLogger,
+    IPathExtension pathExtension,
+    IRelativityAuthenticationProviderFactory relativityAuthenticationProviderFactory)
+    : ISample
+{
+    public async Task ExecuteAsync(Configuration configuration, CancellationToken token)
```

Comment:
This example shows how to upload a file but the sample descriptions suggests how to build a bearer token.
Is it ok?

Reply:
You are right. The class, which implements `IRelativityAuthenticationProvider`, was extracted but it should be here.

Reply:
Done